pkg/podman/podman.go

```diff
@@ -144,6 +184,30 @@ func (o *PodmanCli) VolumeLs() (map[string]bool, error) {
 	return SplitLinesAsSet(string(out)), nil
 }
 
+func (o *PodmanCli) CleanupPodResources(pod *corev1.Pod) error {
```

Comment:
This fails if `odo dev` was stopped because of an error.
```sh
$ ODO_EXPERIMENTAL_MODE=true odo delete component   
============================================================================
⚠ Experimental mode enabled. Use at your own risk.
More details on https://odo.dev/docs/user-guides/advanced/experimental-mode
============================================================================

 •  The following pods and associated volumes will be deleted from podman:
        - my-nodejs-app-app
? Are you sure you want to delete "my-nodejs-app" and all its resources? Yes
 ⚠  Failed to delete the pod "my-nodejs-app-app" from podman: exit status 125: Error: error freeing lock for container dfb4ce8ee86a29b5c8845a2dd8818d801990a40e2413ef6b7876b7c8240d939a: no such file or directory
```

To reproduce:
1. `mkcd /tmp/nodejs`
2. `odo init --devfile nodejs --name my-node-app`; skip downloading the starter project
3. `ODO_EXPERIMENTAL_MODE=t odo dev --run-on podman`; this will fail during the build because there will be no project files.
4. `ODO_EXPERIMENTAL_MODE=t odo delete componen`.

Reply:
I was unable to reproduce this again, it is a glitch with podman, we can ignore it.